keywords/SyncGateway.py

```diff
@@ -561,11 +578,16 @@ def upgrade_sync_gateways(self, cluster_config, sg_conf, sync_gateway_version, u
                 log_info("Keyerror in getting logging{}".format(ex.message))
                 playbook_vars["logging"] = '{} {},'.format(logging_config, "}")
 
-            if get_sg_use_views(cluster_config):
-                playbook_vars["sg_use_views"] = '"use_views": true,'
+            server_version = get_cbs_version(cluster_config)
+            cbs_version, cbs_build = version_and_build(server_version)
+            if cbs_version >= "5.5.0":
+                if get_sg_use_views(cluster_config):
+                    playbook_vars["sg_use_views"] = '"use_views": true,'
```

Comment:
It is same logic even for else. you can combine the logic

Reply:
checked in a new copy, combined the logic

Reply:
done